src/controllers/connection/newConnection.ts

```diff
@@ -107,11 +108,11 @@ export class NewConnectionController extends HTMLController {
   }
 
   /**
-   * @returns a company from a validated connection invitation
+   * @returns a invite from a validated connection invitation
    */
   @SuccessResponse(200)
   @Get('/verify-invite')
-  public async verifyInviteForm(@Query() invite: BASE_64_URL | string): Promise<HTML> {
+  public async verifyInviteForm(@Query() invite: BASE_64_URL): Promise<HTML> {
```

Comment:
This was deliberately an `| string` so that we can provide feedback for invalid base64. Please revert

Reply:
makes sense, added.